contracts/Linea_SpokePool.sol

```diff
@@ -29,7 +30,7 @@ contract Linea_SpokePool is SpokePool {
     /**
      * @notice Address of Linea's USDC Bridge contract on L2.
      */
-    IUSDCBridge public l2UsdcBridge;
+    IUSDCBridge private DEPRECATED_l2UsdcBridge;
```

Comment:
I think because this contract is a Child contract and won't get extended, we can delete this storage slot? However this is risky if this contract ever gets extended by a future Child contract so safer to just make it private and preserve the slot

Reply:
thats an interesting idea. Are there negative side-effects to keeping this as-is ? I can't think of any harm

Reply:
Actually not since you marked it private. I overlooked that on the first pass. So there's basically no benefit to changing it :+1: